quantum/split_common/matrix.c

```diff
@@ -266,7 +280,7 @@ void matrix_post_scan(void) {
 }
 
 uint8_t matrix_scan(void) {
-    bool changed = false;
+    bool changed = matrix_pre_scan();
```

Comment:
I'm not sure yet, but maybe we should do this.
```suggestion
    bool changed = false;
    matrix_pre_scan();
```

I will find time later and consider it. I want you to wait until then.

Reply:
The intention was to be able to determine if the slave-side matrix had changed as well. If we drop the return value, then we no longer get the comparison for the slave-side half.

If you've got another suggestion as far as implementation is concerned, it'd be great.

Reply:
I don't think it is necessary to pass the change information of the key matrix on the slave side to `debounce()` on the master side.

```c
uint8_t matrix_scan(void) {
    bool local_changed = false;
    bool remote_changed = matrix_pre_scan();

    /// local matrix scan...

    debounce(raw_matrix, matrix + thisHand, ROWS_PER_HAND, local_changed);

    matrix_post_scan();
    return (uint8_t)(local_changed || remote_changed);
}
```

Reply:
Yep, fair enough -- made the requested changes.